tests/test_resolutions.py

```diff
@@ -45,19 +45,18 @@ def test_can_be_instantiated(self):
 
         assert isinstance(ic, IsClickable)
 
-    @mock.patch("selenium.webdriver.remote.webelement.WebElement", spec=WebElement)
-    def test_matches_a_clickable_element(self, element):
+    def test_matches_a_clickable_element(self):
+        element = mock.create_autospec(WebElement, instance=True)
```

Comment:
Should we have a `get_mock_element` function for this?

Reply:
We could.  I didn't see it reducing the number of lines of code, but I don't think it would hurt anything.